content/02.introduction.md

```diff
@@ -1,18 +1,19 @@
 ## Introduction
 
-Pediatric brain and spinal cord tumors are the second most common tumors in children after leukemia, yet they represent the leading disease-related cause of death in children [@doi:10.1093/neuonc/noz150]. 
+Pediatric brain and spinal cord tumors are the second most common tumors in children after leukemia, and they represent the leading disease-related cause of death in children [@doi:10.1093/neuonc/noz150]. 
 Five-year survival rates vary widely across different histologic and molecular classifications of brain tumors.
-For example, most high-grade and embryonal tumors carry a universally fatal prognosis while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207]. 
-Despite their relative rarity, the years of potential life lost due to brain tumors in 2009 was estimated at 47,631 years for children and adolescents aged 0-19 in the United States [@doi:10.1159/000093896]. 
-The low survival rates for some tumors are clearly multifactorial but can be explained in part by our lack of understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these entities, and the shortage of drugs specifically labeled for pediatric malignancies. 
+For example, most high-grade and embryonal tumors carry a universally fatal prognosis, while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207]. 
+Moreover, estimates from 2006 suggest that children and adolescents aged 0-19 with brain tumors in the United States have lost an average of 47,631 years of potential life [@doi:10.1159/000093896]. 
```

Comment:
I checked this citation, it's from 2006

Reply:
I actually wonder if this is the citation? https://onlinelibrary.wiley.com/doi/full/10.1002/cam4.410

I don't have full access text to thoroughly check though.

Reply:
YES, you got it! From abstract: 0.31*153,390.4 = 47,551, so we should update.

Reply:
Didn't seem to have a paywall for me. Actually, Table 2 is where the `47,631.5` comes from:
<img width="879" alt="Screen Shot 2022-05-09 at 2 15 05 PM" src="https://user-images.githubusercontent.com/207034/167471845-b8b96d80-a4d2-49c8-9fea-29b6c96add16.png">

Reply:
Great! Will update accordingly!!